src/language-html/syntax-attribute.js

```diff
@@ -60,7 +60,7 @@ function printImgSrcset(value) {
   );
 }
 
-const prefixDelimiters = ["__", "--", "_", "-"];
+const prefixDelimiters = ["__", "--", "_", "-", ":"];
```

Comment:
Tailwind classes often contain `-`, which will have higher priority than `:` if implemented this way. Is this intended?

**Prettier pr-10906**
[Playground link](https://deploy-preview-10906--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEAeAJgSwG4AIwA2AhgM4kC8AOiALbpIBmEEAtAIy51IBGRATu1wAPAo2bDRABz5wGmIQCZcLFtNnyFKkhBpwYAC0xQA5izgEScXAFcoM4vHQB9QqRJOYEJ3IIEnADgAGF31+EmoAPlQAeixsCJAAGhAISRhMaBJkUH4+CAB3AAV+BCyUIgJ8ogBPLOTuPiIwAGs9AGVJJqNjZBg+azhkuBpuOHR0MYAZIhNrImM4ADEIPhoiGHSTZBAia08kkH0YGgIAdUN4Ek6wODbSzHTsB+rtsDIDo0s+GELG4zXkAwKpZkgArEhCABCjRa7SIukmRjggOBgxA4KEbW6BDgAEVrBB4CiLGjOnwvtsjicDtIjDBTph0AZkEFktIIJZTo1JNs1F9sMjkgBHAnwX6pMo7EgsKBwMZjA4yEWYGS-eYAxio5KWGiYXr9NEkbF40XIzUk5IwIjcBlM-TIBSWxqYAjdADCOg1IDgJAArAdrJYACrWspAi0gbADACSUAmsDaYD4mDSAEE420YNUccTLABfPNAA)
<!-- prettier-ignore -->
```sh
--parser html
```

**Input:**
<!-- prettier-ignore -->
```html
<div class="md:foo-1 md:bar-1 xl:foo xl:prefix2 --prefix2--something-else unrelated_class_to_fill_80_chars"></div>
```

**Output:**
<!-- prettier-ignore -->
```html
<div
  class="
    md:foo-1
    md:bar-1
    xl:foo xl:prefix2
    --prefix2--something-else
    unrelated_class_to_fill_80_chars
  "
></div>

```

Reply:
You are right. I wasn't aware of that. I changed the order of delimeters and adjusted the test case.
Thanks!